builder/native.go

```diff
@@ -20,7 +20,7 @@ func NativeBuildCommands(lang string) ([]string, error) {
 var nativeCommandsForLang = map[string]map[string][]string{
 	"darwin": {
 		"rust": {
-			"cargo build --target wasm32-wasi --lib --release",
+			"cargo vendor && cargo build --target wasm32-wasi --lib --release",
```

Comment:
Let's make these separate entries, they're meant to be executed sequentially, and it'll give more informative errors if things fail.

Reply:
They are still executed sequentially; && is shorthand for "and if" meaning it will execute the subsequent command if and only if the first command completes successfully. 

I do not believe this will result in any less logging as we are not redirecting stdout nor stderr

https://pubs.opengroup.org/onlinepubs/9699919799/

> A ';' separator or a ';' or <newline> terminator shall cause the preceding AND-OR list to be executed sequentially; an '&' separator or terminator shall cause asynchronous execution of the preceding AND-OR list.